aten/src/ATen/core/ivalue_inl.h

```diff
@@ -430,20 +430,18 @@ struct C10_EXPORT ivalue::Future : c10::intrusive_ptr_target {
   // Expose the default implementation so that external ones can defer to it.
   static std::vector<std::reference_wrapper<const at::DataPtr>>
   defaultDataPtrExtractor(const at::IValue& value) {
```

Comment:
seems to me that `defaultDataPtrExtractor` is only used by pybind_utils.h, maybe we can move this to `python_ivalue.h` to make it handle the PyObject case as well?